packages/notifications-panel/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@hig/notifications-panel",
+  "version": "0.0.1",
+  "description": "HIG NotificationsPanel",
+  "author": "Autodesk Inc.",
+  "license": "Apache-2.0",
+  "homepage": "https://hig.autodesk.com",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/DynamoDS/hig.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "build/index.js",
+  "module": "build/index.es.js",
+  "files": [
+    "build/*"
+  ],
+  "dependencies": {
+    "@hig/behaviors": "^2.1.0",
+    "@hig/flyout": "^2.2.0",
+    "@hig/icon-button": "^3.1.0",
+    "@hig/icons": "^4.1.0",
+    "@hig/progress-ring": "^2.2.0",
+    "@hig/rich-text": "^2.1.0",
+    "@hig/typography": "^2.1.0",
+    "@hig/utils": "^0.4.1",
+    "emotion": "^10.0.0",
+    "prop-types": "^15.7.1",
+    "react-transition-group": "^4.4.2"
```

Comment:
Are the dependencies list up to date?

Reply:
Checking...

Reply:
@filipeotero To make it clear, I did not mean if the versions are up to date. I mean if there are ones no longer part of dependency list since we had to remove a bunch of code

Reply:
Ok. Actually, the @hig/flyout isn't necessary so I removed it.